ibtk/include/ibtk/FEDataManager.h

```diff
@@ -555,8 +555,33 @@ class FEDataManager : public SAMRAI::tbox::Serializable
      *
      * \note The default value for this string is "coordinates system".
      */
+    IBTK_DEPRECATED(
+        "use FEDataManager::getCurrentCoordinatesSystemName() and FEDataManager::setCurrentCoordinatesSystemName() to "
```

Comment:
@drwells your last commit changed some of these messages from `"use..."` to `"Use..."`. Do you want to make this consistent across the library?

Reply:
Good thing we have some QA :) I'll fix this.

Reply:
I don't care that much --- I just noticed that you changed this (from what had previously been the consistent use of lowercase 😁)